ui/src/components/Publication/SimpleResult/index.tsx

```diff
@@ -9,48 +9,79 @@ type Props = {
     user: Interfaces.User;
 };
 
-const SimpleResult: React.FC<Props> = (props): React.ReactElement => (
-    <div className="w-full rounded border border-transparent bg-white-50 p-3 shadow transition-colors duration-500 dark:border-teal-500 dark:bg-transparent dark:shadow-none">
-        <div className="flex justify-between gap-2">
-            <div className="flex w-fit flex-col flex-wrap gap-3 sm:flex-row sm:items-center sm:gap-2">
-                <span
-                    className={`${
-                        props.publication.currentStatus === 'LIVE'
-                            ? 'text-teal-500 dark:text-teal-300'
-                            : 'text-purple-500 dark:text-purple-300'
-                    } flex items-center gap-2 font-semibold leading-3 transition-colors duration-500`}
-                >
-                    {props.publication.currentStatus === 'LIVE' ? (
-                        <OutlineIcons.ExternalLinkIcon className="inline h-4 w-4 leading-3 text-grey-600 dark:text-teal-500" />
+const SimpleResult: React.FC<Props> = (props): React.ReactElement => {
+    const publicationStatus = (publication: Interfaces.UserPublication, user: Interfaces.User) => {
```

Comment:
This can either be a function outside the component (because currently this function is re-created every time when this component re-renders) and just call it inside the component, or, it can be a useMemo inside the component which computes the status that needs to be shown inside the jsx